core/server/data/schema/commands.js

```diff
@@ -43,6 +43,9 @@ function addTableColumn(tableName, table, columnName) {
     if (columnSpec.hasOwnProperty('defaultTo')) {
         column.defaultTo(columnSpec.defaultTo);
     }
+    if (columnSpec.hasOwnProperty('index') && columnSpec.index === true) {
+        column.index();
+    }
```

Comment:
@kirrg001 Added `column.index` property from knex from this doc - https://knexjs.org/#Schema-alter

Reply:
Ah okay, i didn't see the extra commit 👍

Reply:
Could you now please make two clean commits?

1. commands support for index
2. the actual commit

Thanks 👍

Reply:
Then i can rebase the PR